kratos/includes/kratos_parameters.h

```diff
@@ -5,36 +5,28 @@
 //                   Multi-Physics
 //
 //  License:		 BSD License
-//					 Kratos default license:
-//kratos/license.txt
+//					 Kratos default license: kratos/license.txt
 //
 //  Main authors:    Riccardo Rossi
+//                   Vicente Mataix Ferrandiz
 //
 
-#if !defined(KRATOS_KRATOS_PARAMETERS_H_INCLUDED)
-#define KRATOS_KRATOS_PARAMETERS_H_INCLUDED
 
-// System includes
+#if !defined(KRATOS_KRATOS_PARAMETERS_H_INCLUDED )
+#define  KRATOS_KRATOS_PARAMETERS_H_INCLUDED
 
-#include <iostream>
-#include <sstream>
-#include <string>
-#include <utility>
+// System includes
 
 // External includes
+#include "json/json_fwd.hpp" // Import forward declaration nlohmann json library
```

Comment:
could you explain briefly what exactly you did with this?
I know it is needed to pass Travis, but it would be good to know in case someone wants to update the library

Reply:
Th forward declaration is only a declaration of the classes (just the name), which means that you can work only with references and pointers, so the compiler can manage the memory for that

Reply:
https://en.wikipedia.org/wiki/Forward_declaration